README.md

```diff
@@ -1,27 +1,157 @@
 # 🌟 Azure MCP Server
 
+The Azure MCP Server implements the [MCP specification](https://modelcontextprotocol.io) to create a seamless connection between AI agents and Azure services through:
 
-[![Install with NPX in VS Code](https://img.shields.io/badge/VS_Code-Install_Azure_MCP_Server-0098FF?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=Azure%20MCP%20Server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40azure%2Fmcp%40latest%22%2C%22server%22%2C%22start%22%5D%7D) [![Install with NPX in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-Install_Azure_MCP_Server-24bfa5?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=Azure%20MCP%20Server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40azure%2Fmcp%40latest%22%2C%22server%22%2C%22start%22%5D%7D&quality=insiders)
+- 🔄 Smart JSON communication that AI agents understand
+- 🏗️ Natural language commands that get translated to Azure operations
+- 💡 Intelligent parameter suggestions and auto-completion
+- ⚡ Consistent error handling that makes sense
 
-The Azure MCP Server implements the [MCP specification](https://modelcontextprotocol.io) to create a seamless connection between AI agents and key Azure services like Azure Storage, Cosmos DB, and more.
+Azure MCP Server can be used alone or with the [GitHub Copilot for Azure extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azure-github-copilot) in VS Code. If you're interested in broad developer support across a variety of Azure development scenarios not included in the Azure MCP Server, such as searching documentation on Microsoft Learn, we recommend this extension as well.
 
 > Please note that this project is in Public Preview and implementation may significantly change prior to our General Availability.
 
-## 📑 Table of contents
-1. [🎯 Overview](#-overview)
-2. [🛠️ Currently Supported Tools](#%EF%B8%8F-currently-supported-tools)
-3. [🔌 Installation & Getting Started](#-getting-started)
-4. [🧪 Using Azure MCP Server](#-test-the-azure-mcp-server)
-5. [📝 Troubleshooting Issues](#-troubleshooting)
-6. [👥 Contributing to Azure MCP Server](#-contributing)
+### ⚙️ Getting Started Install Steps
+
+1. Install either the stable or Insiders release of VS Code:
+   * [💫 Stable release](https://code.visualstudio.com/download)
+   * [🔮 Insiders release](https://code.visualstudio.com/insiders)
+2. Install the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and [GitHub Copilot Chat](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) extensions
+3. Install [Node.js](https://nodejs.org/en/download) 20 or later
```

Comment:
they only need to check if npx is in path

Reply:
I can fix that.  this is pre-existing text.